lib/parallel_cucumber/hooks.rb

```diff
@@ -0,0 +1,20 @@
+module ParallelCucumber
+  class Hooks
+
+    @after_batch_hooks ||= []
+
+    class << self
+
+      def after_batch(&proc)
```

Comment:
Please add YARD documentation for this method documenting parameters that would be passed to the proc on call.